docs/content_management/content_api/managing_content.md

```diff
@@ -165,6 +165,35 @@ To change the identifier of the copy, use a [`ContentTypeUpdateStruct`](/api/php
 [[= include_file('code_samples/api/public_php_api/src/Command/CreateContentTypeCommand.php', 87, 93) =]]
 ```
 
+### Finding and filtering content types
+
+You can find content types that match specific criteria by using the `ContentTypeService::findContentTypes()` method.
+This method accepts a `ContentTypeQuery` object that supports filtering and sorting by IDs, identifiers, group membership, and other criteria.
+
+!!! note "Criterions and sort clauses"
+
+    For a full list of available criterions and sort clauses that you can use when finding and filtering content types, see [Content Type Search Criteria](content_type_criteria.md) and [Content Type Search Sort Clauses](content_type_sort_clauses.md) references.
+
+
+The following example shows how you can use the criteria to find content types:
+
+```php hl_lines="11-14"
+[[= include_file('code_samples/api/public_php_api/src/Command/FindContentTypeCommand.php', 17, 41) =]]
```

Comment:
```suggestion
[[= include_file('code_samples/api/public_php_api/src/Command/FindContentTypeCommand.php') =]]
```

I think the example is quite short - and we can use it whole, makes copy pasting and experimenting with this API easier.

I'd also highlight the whole query, not just a part of it.